automates/text_reading/src/test/scala/org/clulab/aske/automates/text/TestUnits.scala

```diff
@@ -76,12 +74,42 @@ class TestUnits extends ExtractionTest {
 
   //todo: check source of this example
   val t8a = "The density of water (ρw) is taken as 1.0 Mg m-3."
-  failingTest should s"extract variables and units from t8a: ${t8a}" taggedAs(Somebody) in {
+  passingTest should s"extract variables and units from t8a: ${t8a}" taggedAs(Somebody) in {
     val desired = Seq(
-      "ρw" -> Seq("Mg m-3")
+      "density of water" -> Seq("Mg m-3") // `density of water` is likely preferred over pw because of the keep longest action; this is an acceptable result
 
     )
     val mentions = extractMentions(t8a)
     testUnitEvent(mentions, desired)
   }
-}
\ No newline at end of file
+
+  // supermaas papers
+  val t9a = "Figure 2 : Performance of the fababean module ( observed versus simulated grain yield in g / m2 ) against test datasets reported by Turpin et al. ( 2003 ) ."
+  passingTest should s"extract variables and units from t9a: ${t9a}" taggedAs(Somebody) in {
+    val desired = Seq(
+      "Performance of the fababean module" -> Seq("g / m2")
```

Comment:
Shouldn't this be "observed versus simulated grain yield" -> "g/m2"?

Reply:
No idea 😆 Probably just grain yield actually :D